scalajvm/app/controllers/api/Quotes.scala

```diff
@@ -1,30 +1,47 @@
 package controllers.api
 
-import helpers.ActionWithTx
+import helpers.{ActionWithTx, Cors, RequestWithSession}
 import models.queries.{QuoteFilter, QuoteOrdering, QuoteQueries}
 import models.data.Quote
 import play.api.mvc._
 import ru.org.codingteam.loglist.dto.QuoteDTO
 import ru.org.codingteam.loglist.QuoteCount
 
 object Quotes extends Controller {
-  def getQuote(id: Long) = ActionWithTx { request =>
+  private def action(responseAction: RequestWithSession[AnyContent] => Result) = ActionWithTx { request =>
```

Comment:
Such a generic name attracts generic code. I suggest to rename it to `corsAction` or similar. Otherwise this function may grow uncontrollably in the future.